File: Core python/Assignment/Assignment 3/q1.py
# 1. Write a program to check if the given number is positive or negative.

# take input n
n = int(input("enter the number : "))

# print output base on condition
if(n > 0):
    print(f'{n} is positive . ')
elif(n<0):
    print(f'{n} is negative')
else:
    print(f'{n} is 0')

File: C to Java/Assignments/Assignment 1/q1.java
import java.util.Scanner;

public class q1 {

    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);

        // take input
	
	int n ;
	System.out.printf("enter the number : ");
	n = sc.nextInt();
	
	// check and print , number is even or odd
	if(n % 2 == 0) 
		System.out.printf("number is even");
		
		else
			System.out.printf("number is odd");

        }
}